webapp/src/pages/EditAvatarPage/EditAvatarPage.tsx

```diff
@@ -105,15 +134,26 @@ export default function EditAvatarPage() {
       <EditableAvatar
         url={
           user.avatarId
-            ? `${AVATAR_EP_URL}/${user.avatarId}`
+            ? imgData.imgSrc || `${AVATAR_EP_URL}/${user.avatarId}`
             : WILDCARD_AVATAR_URL
         }
         XCoordinate={picturePosition.x}
         YCoordinate={picturePosition.y}
-        handleMouseDown={handleMouseDown}
-        handleMouseUp={handleMouseUp}
-        handleMouseMove={handleMouseMove}
+        handleDown={handleDown}
+        handleUp={handleUp}
+        handleMove={handleMove}
+        disabled={imgFile !== null}
       />
+      {status.type !== 'pending' && (
+        <Text
+          variant={TextVariant.PARAGRAPH}
+          color={
+            status.type === 'success' ? TextColor.ONLINE : TextColor.OFFLINE
+          }
+        >
+          {status.message}
+        </Text>
+      )}
       <Button
```

Comment:
The button is too wide: this is not how it was supposed to look in figma. Someone broke the button width at some point in early September.

Reply:
Yes not 1:1 like Figma :smile: Since this is no blocking to merge this, we can review it in future tasks/chat iterations